static/js/i18n/i18n.tsx

```diff
@@ -165,6 +165,10 @@ function formatNumber(value: number, unit?: string): string {
   let shouldAddUnit = false;
   let unitKey: string;
   switch (unit) {
+    case "₹":
+      shouldAddUnit = true;
+      unitKey = "rupee";
```

Comment:
nit: "Indian Rupee"?  Since there are others too (https://en.wikipedia.org/wiki/Rupee).

Also, the currency code equivalent to USD for Indian Rupee is INR.

Reply:
Nice, I changed to use the currency code.